src/components/story/Story.vue

```diff
@@ -1,17 +1,16 @@
 <template>
-  <!-- Carousel Source: https://ismail9k.github.io/vue3-carousel/getting-started.html -->
   <Carousel @slide-end="handleSlideEnd">
-    <Slide v-for="slide in totalSlides" :key="slide">
-      <div v-if="slide === 1" id="sceneOne">
-        <SceneOne :src="imgSrc(slide)" />
+    <Slide v-for="(slide, index) in totalSlides" :key="index">
+      <div v-if="index === 0" id="sceneOne">
+        <SceneOne :src="imgSrc(index + 1)" />
```

Comment:
why do you need the `index` and why stopped using the `slide` variable?

Reply:
i wanted to match the index with image names (1, 2, 3 and so on), slide wasn't working when i tried it locally so i assumed it would need an index to build the path

Reply:
ok, a couple of tips:
1. both the `slide` and the `index` variables are numbers. So changing variable and adding/subtracting 1 from it wont solve any problem
2. whenever you do a change which doesnt solve a problem, always revert that change. It reduces random complexity and logic from building up un-necessarily

Reply:
got it